docker/docker-compose-mock-ai.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.8'
+services:
+  mock-ai:
```

Comment:
Have commented to Peter that mock-ai is now deprecated and not even built by concourse any more.
New JIRA to be created by Peter for immed action to remedy this with a switch for this docker suite to use mock-service instead

Reply:
SOCINT-123 has been created to update rh-ui and rh-service when mock-case is completed.